packages/js/components/src/product-section-layout/product-field-section.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
These two files are copied back over, the only addition is the `deprecated` function call.